pkg/chartverifier/checks/checks_test.go

```diff
@@ -403,28 +412,34 @@ func TestImageParsing(t *testing.T) {
 		expectedVersion  string
 		expectedRepo     string
 		expectedRegistry string
+		expectedSha      string
```

Comment:
we could replace all these with `expectedImageRef *ImageReference`?

Reply:
Done